src/SingleZodiac.js

```diff
@@ -70,29 +71,17 @@ class SingleZodiac extends Component {
                 <p>Lucky Number: {this.state.json.lucky_number} </p>
                 <p>Lucky Time of Day: {this.state.json.lucky_time} </p>
                 <Link to={`/${this.props.id}/yesterday`}>
-                  <button
-                    onClick={(e) => {
-                      this.props.updateDay('yesterday');
-                    }}
-                  >
+                  <button>
                     Yesterday
                   </button>
                 </Link>
                 <Link to={`/${this.props.id}/today`}>
-                  <button
-                    onClick={(e) => {
-                      this.props.updateDay('today');
-                    }}
-                  >
+                  <button>
                     Today
                   </button>
                 </Link>
                 <Link to={`/${this.props.id}/tomorrow`}>
```

Comment:
Too many things happening at once, thank you for getting rid of the Link to so that things could function properly.